chain/jsonrpc/src/lib.rs

```diff
@@ -396,19 +399,34 @@ impl JsonRpcHandler {
             "broadcast_tx_commit" => {
                 process_method_call(request, |params| self.send_tx_commit(params)).await
             }
+            "changes" => {
+                process_method_call(request, |params| self.changes_in_block_by_type(params)).await
+            }
             "chunk" => process_method_call(request, |params| self.chunk(params)).await,
             "gas_price" => process_method_call(request, |params| self.gas_price(params)).await,
+            "genesis_config" => {
+                process_method_call(request, |_params: ()| async {
+                    Result::<_, std::convert::Infallible>::Ok(&self.genesis_config)
+                })
+                .await
+            }
             "health" => process_method_call(request, |_params: ()| self.health()).await,
             "light_client_proof" => {
                 process_method_call(request, |params| {
                     self.light_client_execution_outcome_proof(params)
                 })
                 .await
             }
+            "maintenance_windows" => {
+                process_method_call(request, |params| self.maintenance_windows(params)).await
+            }
             "next_light_client_block" => {
                 process_method_call(request, |params| self.next_light_client_block(params)).await
             }
             "network_info" => process_method_call(request, |_params: ()| self.network_info()).await,
+            "protocol_config" => {
+                process_method_call(request, |params| self.protocol_config(params)).await
```

Comment:
nearcore team does not treat protocol config as stable API, so I refuse to remove EXPERIMENTAL from it until the rules are clarified and nearcore team commits to it.

The recent case study: https://github.com/near-cli-rs/near-validator-cli-rs/issues/23 - `disable_9393_fix` field was just removed from the struct and this is a backwards incompatible change.

Reply:
protocol config changes reverted